<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="redniBroj"/>
      </item>
      <item row="1" column="2" rowspan="12" colspan="8">
       <widget class="QTableView" name="tableView"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Redni broj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="10" rowspan="12" colspan="3">
       <widget class="QTableView" name="tableView_2"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Lista printera</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Prethodno stanje brojača</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Trenutno stanje brojača</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Serijski broj</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="13" column="12">
       <widget class="QLabel" name="label_ukupnaPotrosnja">
        <property name="text">
         <string>Ukupna potrošnja</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Lokacija</string>
        </property>
       </widget>
      </item>
      <item row="14" column="4">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Godina</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="inventurniBroj"/>
      </item>
      <item row="14" column="6">
       <widget class="QPushButton" name="pushButton_sljedeciMjesec">
        <property name="text">
         <string>Sljedeći mjesec</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_update">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item row="14" column="5">
       <widget class="QSpinBox" name="spinBox_godina">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>2023</number>
        </property>
        <property name="maximum">
         <number>2029</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="trenutnoStanje"/>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lokacija"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Model Printera</string>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <widget class="QSpinBox" name="spinBox_mjesec">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="prethodnoStanje"/>
      </item>
      <item row="14" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Mjesec</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Inventurni broj</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="serijskiBroj"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="modelPrintera"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Zadužena Osoba</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="osoba"/>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>Obriši</string>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="label_serijski">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Serijski broštampača:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
